.github/workflows/Documenter.yml

```
@@ -30,6 +30,7 @@ jobs:
         with:
           version: '1.6'
```

```suggestion
          version: '1.10'
```
Should we also update to the latest Julia version such that we can benefit from pkgimage caching?

---

Sounds reasonable